src/components/dataset/overview/connection-to-forms.vue

```diff
@@ -0,0 +1,116 @@
+<!--
+Copyright 2023 ODK Central Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/getodk/central-frontend/blob/master/NOTICE.
+
+This file is part of ODK Central. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+-->
+
+<template>
+  <summary-item id="connection-to-forms" icon="magic-wand">
+    <template #heading>
+      {{ propertiesByForm.length }}
+    </template>
+    <template #body>
+      <p>{{ $t('formsCreateEntities') }}</p>
+      <div class="div">
+        <expandable-row v-for="(form) in propertiesByForm" :key="form.xmlFormId">
+          <template #title>
+            <router-link :to="publishedFormPath(projectId, form.xmlFormId)" :title="form.name">{{ form.name }}</router-link>
```

Comment:
Could you add assertions about this link in testing, of its `to` prop and its `title` attribute? And add another test that the `to` prop is correct if the user is a project viewer?

Reply:
Project viewer also has 'form.read', so link will be same. Instead I have added unit tests in mixins/routes.js for validating `publishedFormPath`